app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,33 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>MembersOnly</title>
+    <title>Posts</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
+  <%= link_to root_path do%>
+    <h1> Home </h1>
+  <% end %>
   <body>
     <%= yield %>
+    <% if user_signed_in? %>
+  <p>
+    <%= link_to current_user.username, edit_user_registration_path %> 
+  </p>
+  <p>
+    <%= link_to "Logout", destroy_user_session_path, method: :delete %> 
+  </p>
+  <% else %>
+  <p>
+    <%= link_to "Log In", new_user_session_path %> 
+  </p>
+  <p>
+    <%= link_to "Sign up", new_user_registration_path %> 
+  </p>
+  <% end %>
```

Comment:
- [ ] put the if/else statement in a helper method that renders it's content instead fo having the if/else statement directly in the view since the content has more than one line you should create 2 partials to put the markup in the if statement and the else statement respectively and render the partials within the helper method

Reply:
Done